android/src/fabric/java/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -105,6 +105,17 @@ class PagerViewViewManager : ViewGroupManager<NestedScrollableHost>(), RNCViewPa
         return PagerViewViewManagerImpl.needsCustomLayoutForChildren()
     }
 
+    @ReactProp(name = "page")
+    override fun setPage(view: NestedScrollableHost?, value: Int) {
+        val animated = PagerViewViewManagerImpl.animated
```

Comment:
I would pass the `PagerViewViewManagerImpl.animated` directly to the function

Reply:
Passed `PagerViewViewManagerImpl.animated` directly